src/main/java/groovy/lang/ExpandoMetaClass.java

```diff
@@ -477,11 +477,11 @@ public static void disableGlobally() {
         ExpandoMetaClassCreationHandle.disable();
     }
 
-    /* (non-Javadoc)
-     * @see groovy.lang.MetaClassImpl#initialize()
+    /**
+     * {@inheritDoc}
      */
-
-    public void initialize() {
+    @Override
+    public synchronized void initialize() {
         try {
```

Comment:
The method implements synchronization via ` lock` API,  the `synchronized` seems redundant.

Reply:
You're right. Removed `synchronized.`